site/content/en/docs/drivers/_index.md

```diff
@@ -31,6 +31,16 @@ To do so, we use the [Docker Machine](https://github.com/docker/machine) library
 * [VMware Fusion]({{<ref "vmware.md">}}) - VM
 * [SSH]({{<ref "ssh.md">}}) - remote ssh
 
+### Apple Silicon Support
```

Comment:
how about we named this  "macOS M1 (Silicon)"

Reply:
Sure sounds good.

Reply:
Changed.

Reply:
Better if this was the same as on any other platform ?

Less Apple, less Silicon, less M1. More good ole "Arch"